media-libs/freetype/freetype-2.7.1.recipe

```diff
@@ -8,6 +8,7 @@ LICENSE="FreeType"
 REVISION="1"
 SOURCE_URI="https://download.savannah.gnu.org/releases/freetype/freetype-$portVersion.tar.bz2"
 CHECKSUM_SHA256="3a3bb2c4e15ffb433f2032f50a5b5a92558206822e22bfe8cbe339af4aa82f88"
+#PATCHES="freetype-2.7.1-subpixel.patchset" # enable patented subpixel rendering
```

Comment:
You can just name it "freetype-subpixel", it should still work on future versions, too.